vulpea-db.el

```diff
@@ -188,71 +188,181 @@ group by
                       (funcall filter-fn note))
               (push note notes))))))))
 
+(defun vulpea-db-query-by-ids (ids)
```

Comment:
This function will stay, and I think I will simply change implementation of `vulpea-db-query` to use this one. I just want to play around with it a little bit.